app/controllers/projects/samples_controller.rb

```diff
@@ -32,11 +32,24 @@ def show
       @tab = params[:tab]
       @table_listing = if @tab == 'metadata'
                          @sample.metadata_with_provenance
+                       elsif @tab == 'history'
+                         @log_data = @sample.log_data_without_changes
```

Comment:
You have @table_listing = @log_data = @sample.log_data_without_changes here. I figure you can either:

Rejig the if else tree whether you want to use @table_listing for consistency or maybe we should specify the passed param for each tab (instead of table_listing, have sample_metadata = and attachments = for metadata and files, respectively).

We could also just ditch the table_listing/log_data/whatever new variable assignment and call @sample for each in the `show.turbo_stream.erb` but figure we may implement a tab in the future that doesn't have data as easily accessible.

Reply:
Whoops thanks for catching this as well. I will fix up the if/else tree so the correct variable is assigned

Reply:
Fixed the assignment of the log_data variable and updated the variable names for files and metadata tab.